src/js/jsx/sections/layers/LayerGroup.jsx

```diff
@@ -112,6 +146,20 @@ define(function (require, exports, module) {
             // which allows us to skip rendering a subtree if its children remain unchanged.
             return true;
         },
+        
+        componentDidUpdate: function () {
+            if (this._selectedLayerHasNewIndex) {
+                this._selectedLayerHasNewIndex = false;
+                this._scrollIntoFirstSelectedLayer();
+            }
+        },
+
+        /**
+         * Scroll into the first selected layer if it is not visible in the layers panel when
+         */
+        _scrollIntoFirstSelectedLayer: function () {
+            window.document.querySelector(".face__selected").scrollIntoViewIfNeeded();
```

Comment:
I guess this is just an aesthetic thing, but would it be possible to put the method that actually performs the scroll into the face component, and then have the group use a child ref to call that method?